Demo/Config/Section.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+class Section: NSObject, UITableViewDataSource {
```

Comment:
How about making `Section` into a simple struct with title and data only. Because `DataSource` is already `UITableViewDataSource` and I think it makes more sense to do table view setup there. What do you think?

Reply:
I agree